scipy/optimize/optimize.py

```diff
@@ -1088,6 +1088,41 @@ def fmin_bfgs(f, x0, fprime=None, args=(), gtol=1e-5, norm=Inf,
     allvecs  :  list
         The value of xopt at each iteration. Only returned if retall is True.
 
+    Examples
+    --------
+    >>> import numpy as np
```

Comment:
This import is implicit in our docstrings so can be removed (though it has caused some controversy recently because it is easier to copy-paste with an explicit import).

Reply:
Thank you for your feedback.